editor/src/components/canvas/canvas-strategies/strategies/flow-reorder-strategy.spec.browser2.tsx

```diff
@@ -228,9 +351,10 @@ async function startDraggingAnElement(
   const canvasControlsLayer = renderResult.renderedDOM.getByTestId(CanvasControlsContainerID)
 
   const startPoint = windowPoint({ x: targetElementBounds.x + 5, y: targetElementBounds.y + 5 })
+  const endPoint = offsetPoint(startPoint, dragDelta)
 
-  await mouseDownAtPoint(canvasControlsLayer, startPoint, {})
-  await mouseMoveToPoint(canvasControlsLayer, dragDelta, {})
+  await mouseDownAtPoint(canvasControlsLayer, startPoint, { modifiers: cmdModifier })
+  await mouseMoveToPoint(canvasControlsLayer, endPoint, {})
```

Comment:
We weren't selecting the actual element here, or using the `dragDelta` as a delta, so the tests using this function were only working by pure coincidence (since they were checking that a strategy wasn't applicable, which it wasn't!)